### ISSUE TITLE
I can teach quantum physics to your dog 

### ISSUE THREAD

**Issue by davidwesst** [davidwesst/blog#19]
I finished _How to teach physics to your dog_ by Chad Orzel as I continue to dive deeper into my quest in understanding in quantum computing. This book was recommended by Mark Russonovich at the end of his Microsoft Ignite 2019 talked called _Quantum computing: Computing with a probabilistic universe with Mark Russinovich_ at Microsoft Ignite 2019. You can watch at this link at the Microsoft Ignite site: [LINK](https://myignite.microsoft.com/archives/IG19-BRK4011).

In short, it's pretty good.

It's solid way to understand some of the core of quantum mechanics that make quantum computing possible. It goes deeper just defining concepts like superposition, entanglement, and QED, but it also gets into a bit of the math and the history side of the concepts.

# The Highlights
Now, I'm no book scientist and as such, I'm not going to worry about trying to put together a fancy book review that rates my experience. Instead, I thought it would be a good idea to highlight the parts of the book that made it a good read, and you can figure out the rest on your own.

## Dog Conversations
Orzel jumps between explaining a concept traditionally to having a conversation with his dog (hense the title). 

The style is kinda silly, which is something of a fresh take for explaining science, but it works surprisingly well. 

Outside of helping to simplify some of the more complicated or mind bending points of quantum mechanics, or lightens the mood and makes it less of a "physics book" and more of a "story about physics".

## A story that teaches you about physics
I think the best part about the book is how it builds on each topic, chapter by chapter. 

It reads like a story, where each chapter prepares you for the next until you finally hit the big wrap up on entanglement and QED, or quantum electrodynamics, along with a chapter on what to look for when people are trying to abuse quantum physics to push their own non-scientific agenda of making a quick buck.

## Mathy, but not too mathy
Physics is math. That's just how it is, and this book does a great job of introducing us to the math, but not dwelling or depending on it. I appreciated that, especially near the end of the book where ideas like QED start to really bend your mind.

I appreciated how they didn't shy away from it and used the conversations with the dog to help bring it down to a "not too mathy" way. Keeps the reader in check and reminds them that this isn't just philosophy, but _real science_.

# TL;DR; / Conclusion

This book set me up with a solid foundation on the physics that make quantum computing possible, which was the whole reason for reading it. It's "learning through story and conversation" approach that doesn't shy away from maths (but doesn't dwell on them) makes it an easier read than one might think.

Recommended for those interesting in understanding the _science_ (not magic) of quantum physics that make quantum computing a real life thing.

**Comment by davidwesst** [comment]
closed with #20
